ringhold/README.md

```diff
@@ -18,17 +18,25 @@ org.opencadc.ringhold.inventory.schema={schema for inventory database objects}
 org.opencadc.ringhold.inventory.username={username for inventory admin}
 org.opencadc.ringhold.inventory.password={password for inventory admin}
 org.opencadc.ringhold.inventory.url=jdbc:postgresql://{server}/{database}
+
+# artifact namespace
```

Comment:
make the comment `artifact namespace(s) to remove`

Reply:
Updated.